src/sentry/tagstore/types.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+from sentry.tagstore.base import TagKeyStatus
+
+
+class TagKey(object):
```

Comment:
I initially used `namedtuple`s for this but they weren't being serialized correctly (they were being serialized as sequences.) 😞 

I can/was planning on updating these to add `__slots__` and equality/hashing methods, and am OK with doing that either as part of this change or a later one.